insights/client/apps/aws/__init__.py

```diff
@@ -0,0 +1,121 @@
+import logging
+import base64
+from requests import ConnectionError, Timeout
+from requests.exceptions import HTTPError
+from ssl import SSLError
+from urllib3.exceptions import MaxRetryError
+from insights.client.connection import InsightsConnection
+from insights.client.schedule import get_scheduler
+from insights.client.constants import InsightsConstants as constants
+from insights.client.utilities import write_to_disk
+
+logger = logging.getLogger(__name__)
+net_logger = logging.getLogger('network')
+
+IDENTITY_URI = 'http://169.254.169.254/latest/dynamic/instance-identity'
```

Comment:
This isn't a publicly routeable IP, and I can't see where it gets updated to a routeable IP address. Is there some external factor that modifies this variable at runtime?

Reply:
Yeah, check this: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instance-identity-documents.html

Reply:
Ah ha, so it's an AWS thing. Perfect.